### DISCUSSION TITLE
Aula "CRUD"  Acesso à dados com .NET, C#, Dapper e SQL Server.

### DISCUSSION THREAD

**Discussion by RenatoSN12** [balta-io/forum#499] [Tira Dúvidas] [answered]
Após escrever o código conforme a aula explica, meu código está disparando essa exception invés de fazer o Insert do usuário na tabela User.

Unhandled exception. Microsoft.Data.SqlClient.SqlException (0x80131904): Incorrect syntax near the keyword 'User'.

Se eu fazer uma tentativa de alterar o nome da tabela para BlogUser (e o nome da classe também) ai funciona... Será que agora não pode mais ter uma tabela com esse nome? Ou fiz algo de errado no código?
![print1](https://github.com/balta-io/forum/assets/126295531/a1c1762c-9000-4574-989f-99103aa0d5b8)
![print2](https://github.com/balta-io/forum/assets/126295531/152d6552-2992-4927-b9ed-08bb3dc09db4)
![print3](https://github.com/balta-io/forum/assets/126295531/f1bce588-d2c2-45d9-8668-70a064f0f186)
![print4](https://github.com/balta-io/forum/assets/126295531/47c96451-48a1-456a-b9f8-79f64c0df9f5)

**Comment by juliobacoli** [comment, marked as answer]
Normalmente se esse reclama de keyword é pq o C# entende como palavra reservada, mude o nome da tabela e seja feliz :)

**Reply by malighieri** [reply]
É uma questão que o Dapper não resolve? Eu uso o EF e ele consegue lidar com tabelas que tenham nomes com palavras reservadas como User. Meu caso é justamente esse. Tenho uma tabela no sistema que se chama User. E o SQL consegue lidar com ela desde que usada entre aspas duplas que é o que o EF faz ao montar as consultas e comandos.
